core/store/orm/advisory_locks.go

```diff
@@ -0,0 +1,6 @@
+package orm
```

Comment:
Might be easier to put it in models? I can imagine this one causing some import headaches.

Reply:
Yeah, could see that happening